ytmusicapi/navigation.py

```diff
@@ -26,6 +26,7 @@
 ]
 WATCH_VIDEO_ID = ['watchEndpoint', 'videoId']
 NAVIGATION_VIDEO_ID = ['navigationEndpoint'] + WATCH_VIDEO_ID
+NAVIGATION_VIDEO_ID_2 = ['menu','menuRenderer','items',0,'menuServiceItemRenderer','serviceEndpoint','queueAddEndpoint','queueTarget','videoId']
```

Comment:
Please use the existing navigation paths MENU_ITEMS, MENU_SERVICE

Reply:
Done. Please review.